File: solutions/uncategorized/aplusb.java
import java.util.*;
public class Main //main class
{
	public static void main(String[] args) { //main method that runs when the program starts
	    Scanner sc = new Scanner(System.in); //initializes input scanner
	    int n = sc.nextInt(); //scans for and initializes variable n (number of additions)
	    for(int i = 0; i<n; i++) { //loops n times
    	    int a = sc.nextInt(); //scans for and initializes a
    	    int b = sc.nextInt(); //scans for and initalizes b
    		System.out.println(a+b); //prints the sum of a and b
	    }
	}
}

File: solutions/uncategorized/aplusb.py
n = int(input()) #initializes n and scans it from input
for x in range(n): #loops n times
    a = input().split(" ") #scans a line of input, and splits it into an array a
    print(int(a[0])+int(a[1])) #prints the sum of the first and second index of the array a
